src/page-analysis/background/get-fav-icon.js

```diff
@@ -1,20 +1,26 @@
+import favicon from 'favicon'
```

Comment:
Does the favicon module fetch the whole page again to read the url of its icon? Should be an unnecessary round trip (at least with pages that don't cache by default) if we are fetching the page anyway.

Fetching the DOM once and reusing it would however require a bit of reorganisation in the analysis code. May be worth considering though; we had to do a similar thing for reusing a loaded pdf to get both the metadata and extracted text from it.